pycon/settings.py

```diff
@@ -25,7 +25,21 @@
 
 SITE_DATA_ROOT = DATA_DIR + '/site'
 
-DATABASES = {
+DATABASE_TYPE = os.environ.get('DATABASE')
+
+if DATABASE_TYPE == "postgres":
+    user = os.environ.get('DATABASE_USER')
+    password = os.environ.get('DATABASE_PASSWORD')
+    DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'ep2015_testing',
```

Comment:
+1 on adding an env var in this PR

that said I find having all those mandatory env vars quite annoying... I'd prefer have a "default" set to a sqlite db type (somethign like `DATABASE_TYPE = os.environ.get('DATABASE', 'sqlite')) 
Maybe something for a future PR/discussion